### PR TITLE
Update Cloudscale installation to use OCP 4.7 

### PR DESCRIPTION
**Pull request by corvus-ch**
Relating appuio/terraform-openshift4-cloudscale#17

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc [hunk 1]
```diff
@@ -123,6 +125,74 @@ mc mb --ignore-existing \
   "${CLUSTER_ID}/${CLUSTER_ID}-bootstrap-ignition"
 ----
 
+=== Upload Red Hat CoreOS image
+
+. Export the Athorization header for the Cloudscale API.
```
Comment (srueg) [inline docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:130]:
```suggestion
. Export the Authorization header for the Cloudscale API.
```

**Review by srueg — COMMENTED** [review]
Should we maybe introduce a variable to replace all the `4.7` ocurences?
This way it would be simpler to update it in the future...

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc [hunk 2]
```diff
@@ -123,6 +125,74 @@ mc mb --ignore-existing \
...
+
+[source,console]
+----
+export AUTH_HEADER="Authorization: Bearer ${CLOUDSCALE_TOKEN}"
+----
+
+[NOTE]
+====
+The variable `CLOUDSCALE_TOKEN` could be used directly.
+Exporting the variable `AUTH_HEADER` is done to be compatible to the https://www.cloudscale.ch/en/api/[Cloudscale API documentation].
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:140]:
```suggestion
Exporting the variable `AUTH_HEADER` is done to be compatible with the https://www.cloudscale.ch/en/api/[Cloudscale API documentation].
```

**Review by ccremer — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc [hunk 3]
```diff
@@ -123,6 +125,74 @@ mc mb --ignore-existing \
...
+. Check if image already exists
+
+[source,console]
+----
+curl -sH "$AUTH_HEADER" https://api.cloudscale.ch/v1/custom-images | jq -r '.[] | select(.slug == "rhcos-4.7") | .href'
+----
+
+[NOTE]
+====
+If a URL is printed to the output, you can skip the next steps and directly jump to the next section.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:152]:
```suggestion
If an URL is printed to the output, you can skip the next steps and directly jump to the next section.
```

**Review by simu — APPROVED** [review]

**Comment by tobru** [comment]
> Should we maybe introduce a variable to replace all the 4.7 ocurences?

I would appreciate to get that, yes. Good idea.

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc (hunk 3 above)
Comment (corvus-ch) [reply docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:152]: No, Microsoft thinks otherwise. I had it that way but then the vale check failed.

**Review by srueg — APPROVED** [review]

**Review by tobru — APPROVED** [review]

**Pull request merged**